Assets/HoloToolkit/SpectatorView/Plugins.meta

```diff
@@ -0,0 +1,9 @@
+fileFormatVersion: 2
```

Comment:
Once the `WSA.meta` mentioned below is gone, this one will need to be deleted too, as the folder will be empty.